.drone.yml

```diff
@@ -0,0 +1,37 @@
+pipeline:
+  build_6:
+    image: node:6
+    commands:
+      - npm --loglevel warn install
+      - npm run lint
+      - npm run test
+    when:
+      event: [push, pull_request]
```

Comment:
this will run against . basically so the tests in file-vault. we use this pattern everywhere